tests/test_parser.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+import warnings
+from pathlib import Path
+
+import rdflib
+
+_TEST_DIR = Path(__file__).resolve().parent
+
+
+class TestGraphParsing(unittest.TestCase):
+    def setUp(self):
+        self.oxi_graph = rdflib.Graph(store="Oxigraph")
+        self.memory_graph = rdflib.Graph()
+
+    def remove_triples_from_oxi_graph(self):
```

Comment:
Instead of having these stateful methods, what about just creating a new `Graph` per test? This can avoid concurrency errors if we run tests in parallel at some point

Reply:
NP!